Cargo.toml

```diff
@@ -87,6 +87,7 @@ hnswlib = { version = "0.8.1", git = "https://github.com/chroma-core/hnswlib.git
 reqwest = { version = "0.12.9",  features = ["rustls-tls-native-roots", "http2"], default-features = false }
 random-port = "0.1.1"
 ndarray = { version = "0.16.1", features = ["approx"] }
+humantime = { version = "2.2.0" }
```

Comment:
this was already in our dependency tree from object-store

Reply:
Thanks for checkin!